unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/WebRequest/Interfaces/WebRequestAsyncOperation.cs

```diff
@@ -49,6 +49,8 @@ public WebRequestAsyncOperation(UnityWebRequest webRequest)
             disposeOnCompleted = true;
         }
 
+        public void SetNewWebRequest(UnityWebRequest newRequest) { webRequest = newRequest; }
```

Comment:
could we set this as `internal`?
If we keep it `public` it must handle internally the scenario of replacing the `webRequest` like `WebRequestController` does when calling `resultOp.Dispose()` before retrying

Reply:
You're right! done in https://github.com/decentraland/unity-renderer/pull/949/commits/632c6957d2cbfa7703abe3fba3c0b150911bce68. Thanks!